packages/rich-text/src/register-format-type.js

```diff
@@ -119,6 +120,14 @@ export function registerFormatType( name, settings ) {
 
 	dispatch( 'core/rich-text' ).addFormatTypes( settings );
 
+	const emptyArray = [];
+	const getFunctionStackMemoized = memize( ( previousStack = emptyArray, newFunction ) => {
```

Comment:
Would it work to keep the cache size to one?

Reply:
Also, why do both this and `prepareEditableTree` need to be memoized? Could use some more clarification.

Reply:
Probably not as we need one per RichText I think.

Reply:
I don't think so, because a different function could be returned based on `richTextIdentifier`/`blockClientId`.

Reply:
Just wondering because it's probably best to keep cache size as small as possible. I'm guessing the issue is just consecutive calls and nothing else.

Reply:
So how big is this cache going to grow?

Reply:
No way to cache per instance?

Reply:
That's a good question though